src/quo/components/wallet/swap_input/style.cljs

```diff
@@ -75,3 +75,19 @@
 
 (def fiat-amount
   {:color colors/neutral-50})
+
+(def gradient-start
+  {:width    64
+   :position :absolute
+   :top      0
+   :left     0
+   :bottom   0
+   :z-index  1})
+
+(def gradient-end
+  {:width    64
+   :position :absolute
+   :top      0
+   :right    0
+   :bottom   0
+   :z-index  1})
```

Comment:
Maybe we can add a `def` named `gradient-common`  for all the shared keys?

Reply:
Thanks for the suggestion, updated :)